test/storage-based/RedstoneData.test.ts

```diff
@@ -15,6 +15,10 @@ const toBytes32 = ethers.utils.formatBytes32String;
 const serialized = function (x: number): number {
   return x * 10 ** 8;
 };
+function checkPricesDeviation(value1: number, value2: number, maxPercentageDiff: number = 5) {
+  const percentDiff = 100 * Math.abs(1 - value1 / value2);
```

Comment:
Doesn't it depend on the ordering of values. For example lest take 100 and 10
1) 100 * abs(1-10) = 900
2) 100 * abs(1-0.1) = 90

Reply:
I think it could be good enough to treat one number as a base value and document it.

Reply:
Fixed